src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -127,7 +126,7 @@ public class Arm extends SubsystemBase {
 
 	private boolean armEnabled = true;
 
-    private PowerPanel m_powerPanel;
+    private PowerPanel m_powerPanel = new PowerPanel();
 
     public Arm() {
```

Comment:
No. No. No.. and No.

The PDH feeds power to all subsystems.

Reply:
I agree, but we also shouldn't have to pass it as a parameter to every single subsystem. I will remove this.

Reply:
Subsystems should not be passed into other subsystems.